UPDATING.md

```diff
@@ -41,6 +41,10 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Remove airflow.utils.file.TemporaryDirectory
+
+Removing this because the same functionality is provided by `tempfile.TemporaryDirectory`.
```

Comment:
Can you add more information here?
In my opinion, a good note should contain the following information:
[ ] Previous behaviors
[ ] New behaviors
[ ] If possible, a simple example of how to make a change
[ ] If possible, the benefit for the user after migration e.g. "we want to make these changes to unify class names."
[ ] If possible, the reason for the change e.g. reference for AIP-21

Here is a similar example that includes most if not all of the requirements:
 https://github.com/apache/airflow/blob/master/UPDATING.md#removal-of-redirect_stdout-redirect_stderr

Reply:
Are those bullets points somewhere available?

Reply:
No. This is based on my experience. I would expect the following points, so I try to include them.

Reply:
Maybe it's worth to add them to UPDATING.md? ;)

Reply:
If you are interested in the subject, I invite you to read the document below, which contains some good examples.
https://developers.google.com/style/inclusive-documentation

> Not recommended: Before launch, give everything a final sanity-check.
> Recommended: Before launch, give everything a final check for completeness and clarity.

Reply:
I think it's worth doing. I'll do it in a moment.

Reply:
https://github.com/apache/airflow/pull/6960